programs/sbf/rust/iter/src/lib.rs

```diff
@@ -2,9 +2,9 @@
 
 #![allow(clippy::arithmetic_side_effects)]
 
-extern crate solana_program;
-use solana_program::{
-    custom_heap_default, custom_panic_default, entrypoint::SUCCESS, log::sol_log_64,
+use {
+    solana_program_entrypoint::{custom_heap_default, custom_panic_default, SUCCESS},
+    solana_sysvar::program_stubs::sol_log_64,
```

Comment:
This should keep using `solana_program` unfortunately -- the stubs don't ever use the syscall. Unless that's ok